script/zonemaster_backend_rpcapi.psgi

```diff
@@ -30,34 +28,13 @@ use Zonemaster::Backend::Metrics;
 
 local $| = 1;
 
-# Returns a Log::Any-compatible log level string, or throws an exception.
-sub get_loglevel {
-    my $value = $ENV{ZM_BACKEND_RPCAPI_LOGLEVEL} || 'warning';
-    for my $method ( logging_methods(), logging_aliases() ) {
-        if ( $value eq $method ) {
-            return $method;
-        }
-    }
-    die "Error: Unrecognized ZM_BACKEND_RPCAPI_LOGLEVEL $value\n";
-}
-
 Log::Any::Adapter->set(
-    'Dispatch',
-    dispatcher => Log::Dispatch->new(
-        outputs => [
-            [
-                'Screen',
-                min_level => get_loglevel(),
-                stderr    => 1,
-                newline   => 1,
-                callbacks => sub {
-                    my %args = @_;
-                    $args{message} = sprintf "%s [%d] %s - %s", strftime( "%FT%TZ", gmtime ), $PID, uc $args{level}, $args{message};
-                },
-            ],
-        ]
-    ),
+    '+Zonemaster::Backend::Log',
+    log_level => lc $ENV{ZM_BACKEND_RPCAPI_LOGLEVEL},
+    json => $ENV{ZM_BACKEND_RPCAPI_LOGJSON},
```

Comment:
I guess these aren't mentioned anywhere. Maybe we should mention them somewhere so people can find them? We already have docs/Configuration.md for the config file. Maybe we should have one for environment variables as well?